Lib/json/tool.py

```diff
@@ -25,24 +37,36 @@ def main():
                         help='a JSON file to be validated or pretty-printed')
     parser.add_argument('outfile', nargs='?', type=argparse.FileType('w'),
                         help='write the output of infile to outfile')
+    parser.add_argument('--no_escape', action='store_true', default=False,
+                        help='Do not set ensure_ascii to escape non-ASCII characters')
```

Comment:
@methane are you suggesting:

```python
parser.add_argument('--ensure_ascii', action='store_false', default=True)
# And then when calling `json.dump`:
ensure_ascii=options.ensure_ascii
```

The problem here is that it's counterintuitive for `--ensure_ascii` to result in `ensure_ascii=False`. In https://bugs.python.org/issue27413, the discussion settled on `--no-ensure-ascii`, which I'm happy to switch to:

```python
parser.add_argument('--no-ensure-ascii', action='store_true', default=False)
# And then when calling `json.dump`:
ensure_ascii=not options.no_ensure_ascii
```

Just let me know what's preferred.